src/redux/appReducer.js

```diff
@@ -0,0 +1,23 @@
+import * as buttonLogic from '../pressButtonLogic';
+import * as actions from './actionTypes';
+export default function appReducer(
```

Comment:
Please add an empty line after the imports. Let the code breath, it is easier to read that way.